packages/grafana-data/src/transformations/standardTransformersRegistry.ts

```diff
@@ -28,8 +28,27 @@ export interface TransformerRegistryItem<TOptions> extends RegistryItem {
    * React component used as UI for the transformer
    */
   editor: React.ComponentType<TransformerUIProps<TOptions>>;
+
+  /**
+   * Set of categories associated with the transformer
+   */
+  categories?: Set<TransformerCategory>;
+
+  /**
+   * Image path for the transformer
+   */
+  image?: { dark: string; light: string };
 }
 
+export type TransformerCategory =
+  | 'combine'
+  | 'calculateNewFields'
+  | 'createNewVisualization'
+  | 'filterAndReduce'
```

Comment:
seems like two very different things to form a category :)

Reply:
Renamed that one to just "filter" as there were no reduced involved